python/cudf/cudf/_libxx/lib.pxd

```diff
@@ -153,3 +154,4 @@ cdef extern from "<utility>" namespace "std" nogil:
     cdef unique_ptr[table] move(unique_ptr[table])
     cdef vector[unique_ptr[column]] move(vector[unique_ptr[column]])
     cdef device_buffer move(device_buffer)
+    cdef table_with_metadata move(table_with_metadata)
```

Comment:
I think you should define this inside the `io.types.pxd` instead of here so that we don't need to import it in `lib.pxd`